src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -56,4 +56,7 @@ THE SOFTWARE.
     <f:entry title="${%Skip marking build as unstable on test failure}" field="skipMarkingBuildUnstable">
             <f:checkbox default="false" title="${%If checked, the test failures will still be reported but won't mark the build as unstable}"/>
     </f:entry>
+    <f:entry title="${%Include report files older than build start}" field="parseOldReports">
+            <f:checkbox default="false" title="${%If checked, the test report files older than build start will be included in the parsing}"/>
```

Comment:
this implies that it is defaulting to the poor behaviour of before?

I expected the other way around, see https://github.com/jenkinsci/junit-plugin/pull/367#issuecomment-1122369873

Reply:
I guess this behaviour just makes everybody happy.
Those who are using static nodes will have same behaviour as before and those who are using temporary node will not have problem neither.

Reply:
I haven't read the complete changeset but I guess what you're implying (not described in the PR):

There's no longer any error thrown that was thrown in the previous previous behaviour but it defaults to not showing 'cached' or 'left over' test results.

cc @jglick

Reply:
> I haven't read the complete changeset but I guess what you're implying (not described in the PR):
> 
> There's no longer any error thrown that was thrown in the previous previous behaviour but it defaults to not showing 'cached' or 'left over' test results.
> 

any unit test for that?

Reply:
No as it was the absence of a feature it didn’t make sense to test that to me.

the previous behaviour wasn’t even tested in the plugin

Reply:
well frankly if a majority think removing a long existing behaviour and/or change it as non default makes sense.
why not... I do not have strong argument. I just want to give users the opportunity to have the same behaviour as it used to be.

Reply:
FWIW my advice would be for the simpler behavior of #348 to be the default, and you would need to opt into this special workaround for misconfigured build tools.